metapackage_dependencies.rosinstall

```diff
@@ -4,3 +4,7 @@
 
 # When this file starts getting used, add it to the options (TARGET_WORKSPACE: "$TARGET_REPO_PATH metapackage_dependencies.rosinstall") of the following CI stages: arm64_build , amd64_build, clang-tidy, catkin_lint
 
+- git:
+    uri: https://github.com/uwrobotics/uwrt_mars_rover_hw_bridge.git
+    local-name: uwrt_mars_rover/uwrt_mars_rover_hw_bridge_pkg/uwrt_mars_rover_hw_bridge/
```

Comment:
you shouldn't use a package suffix, it is redundant (and should be avoided as per REP 144). I think you should just call the package `uwrt_mars_rover_hw_bridge` and have the submodule be under `uwrt_mars_rover_hw_bridge/lib/uwrt_mars_rover_hw_bridge`